### ISSUE TITLE
Allow configurable BIP32 derivation paths

### ISSUE THREAD

**Issue by justinmoon** [justinmoon/junction#12]

**Comment by dasource** [comment]
FYI - below paths should give fairly good compatibility with other multisig implementations.

Legacy (P2SH) m/45'/0/0/0
Segwit Compatibility (P2WSH-P2SH) m/48'/0'/0'/1' 
Native Segwit (P2WSH) m/48'/0'/0'/2'

**Comment by justinmoon** [comment]
Agreed

**Comment by Sjors** [comment]
I wouldn't necessarily make them configurable, but they definately should follow the standard. In addition to what @dasource points out for address type, receive should use `...0/n` and change should use `.../1/n`. It currently uses 0 for both.

It does seem to correctly get "married" paths, i.e. the same derivation for each device. You may also want to sort by fingerprint. I'm not at all up to date on good conventions though. Maybe @instagibbs has some ideas?

**Comment by justinmoon** [comment]
@Sjors Correct. I haven't implemented change addresses yet. Configurable BIP32 paths would really only be useful if Junction is used as a recovery tool. Definitely not a priority.
